.gitignore

```diff
@@ -1 +1,2 @@
-.idea
\ No newline at end of file
+.idea
```

Comment:
Should not be needed. You can just have a `.gitignore` in your home dir to exclude such dirs for all projects.